src/test/java/com/atlassian/db/replica/spi/DefaultReplicaConnectionPerUrlProvider.java

```diff
@@ -0,0 +1,20 @@
+package com.atlassian.db.replica.spi;
```

Comment:
Is it SPI or API?

Reply:
It's a naive implementation used only in tests so I guess it's neither `spi` nor `api` (maybe `internal` then?).
I could make it a default implementation and move it to the `src/main/.../api` package
@wyrzyk WDYT?

Reply:
Make it internal please.

Reply:
> I could make it a default implementation

Never mind, I just saw the next comment (about this implementation not working in prod). 
I will move it to `src/test/.../internal` then